src/components/Synthetics/ExecutionPriceRow.tsx

```diff
@@ -0,0 +1,164 @@
+
+interface Props {
+  tradeFlags: TradeFlags;
+  displayDecimals?: number;
+  fees?: TradeFees;
+  executionPrice?: bigint;
+  acceptablePrice?: bigint;
+  triggerOrderType?: OrderType.LimitDecrease | OrderType.StopLossDecrease;
+}
+
+export function ExecutionPriceRow({
+  fees,
+  executionPrice,
+  tradeFlags,
+  acceptablePrice,
+  triggerOrderType,
+  displayDecimals,
+}: Props) {
+  const { isLimit, isMarket, isIncrease, isLong, isTrigger } = tradeFlags;
+
+  const triggerThresholdType = useMemo(() => {
+    if (isIncrease) {
+      return isLong ? TriggerThresholdType.Below : TriggerThresholdType.Above;
+    } else {
+      if (triggerOrderType === OrderType.LimitDecrease) {
+        return isLong ? TriggerThresholdType.Above : TriggerThresholdType.Below;
+      }
+      if (triggerOrderType === OrderType.StopLossDecrease) {
+        return isLong ? TriggerThresholdType.Below : TriggerThresholdType.Above;
+      }
+    }
+  }, [isIncrease, isLong, triggerOrderType]);
+
+  const fullPositionPriceImpactBps =
+    fees?.positionPriceImpact?.bps !== undefined && fees?.priceImpactDiff?.bps !== undefined
+      ? fees?.positionPriceImpact?.bps + fees?.priceImpactDiff?.bps * -1n
+      : undefined;
+
+  const positionPriceImpactDeltaUsd =
+    fees?.positionPriceImpact?.deltaUsd !== undefined && fees?.priceImpactDiff?.deltaUsd !== undefined
+      ? fees?.positionPriceImpact?.deltaUsd + fees?.priceImpactDiff?.deltaUsd * -1n
+      : undefined;
+
+  const acceptablePriceClarification = useMemo(() => {
+    if (isMarket) {
+      return t`The order's acceptable price includes the current price impact and set allowed slippage. The execution price must meet this condition for the order to be executed.`;
+    }
+
+    if (isLimit) {
+      return (
+        <>
+          {isLong ? (
+            <Trans>
+              Once the limit price hits the mark price, the order will attempt to execute, guaranteeing the acceptable
+              price. In the case of negative price impact, the mark price may need to be lower than the limit price.
+            </Trans>
+          ) : (
+            <Trans>
+              Once the limit price hits the mark price, the order will attempt to execute, guaranteeing the acceptable
+              price. In the case of negative price impact, the mark price may need to be higher than the limit price.
+            </Trans>
+          )}
+          <br />
+          <br />
+          <Trans>
+            The order's acceptable price includes the set acceptable price impact. The execution price must meet this
+            condition, and there must be sufficient liquidity for the order to be executed.
+          </Trans>
+        </>
+      );
+    }
+
+    if (isTrigger) {
+      if (triggerOrderType === OrderType.LimitDecrease) {
+        return t`The order's acceptable price includes the set acceptable price impact. The execution price must meet this condition for the order to be executed.`;
```

Comment:
just caught my eye
in same scope you're using both `Trans` and `t`, let's use only one of those?

Reply:
Makes sense